user-interface/src/main/java/life/qbic/datamanager/views/login/passwordreset/ResetEmailSentComponent.java

```diff
@@ -0,0 +1,42 @@
+package life.qbic.datamanager.views.login.passwordreset;
+
+import com.vaadin.flow.component.ClickEvent;
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.spring.annotation.SpringComponent;
+import com.vaadin.flow.spring.annotation.UIScope;
+import java.io.Serial;
+import life.qbic.datamanager.views.general.CardLayout;
+
+/**
+ * Reset Email Sent Component
+ * <p>
+ * Card Stylized component similar to {@link com.vaadin.flow.component.login.LoginOverlay}
+ * component. Informing the user that a reset password email was sent and directing her to the login
+ * layout
+ */
+@SpringComponent
+@UIScope
+public class ResetEmailSentComponent extends CardLayout {
```

Comment:
why not make this a page instead and rout to it from the reset page?

Reply:
That's a good question. 
Currently there is not a real benefit to seperate this functionality into a different page, since the user will only be shown the reset email sent component if he triggered the process during the password reset main. Do you think there is a need to seperate this functionality into different routes?

Reply:
I think that this information has nothing to do with the reset form. No strong feelings just feels like an independent informational page.

Reply:
I'd like to keep it like this for now if that's ok. Down the line if the need arises, it's easily changed to a seperate page 👍